README.md

```diff
@@ -1,33 +1,115 @@
+
+<!-- Usage Guidelines -->
+### Usage Guidelines 
 
 - The converter page contains two editors, one for writing the source code that you want to convert and the other one which will display the
 pseudocode after the conversion.
 - You can also upload the file to be converted by firstly choosing the language from the dropdown menu and then browsing the file and uploading it.
 - The file that you chose will also be displayed in the text editor.
 - Now, just click on the convert button, it will run the docker container based compiler in the background.
 
-## Future Scopes
-- Add support to more languages
-- Convert one language code into other like cpp code to php and so on.
-- Improving the algorithm of the translator to include complex cases like multiple functional calls inside the main.
+### Built With
+
+- [Python](https://www.python.org/)
+- [Django](https://www.djangoproject.com/)
+- [React](https://reactjs.org/)
+- [Dfinity](https://dfinity.org/)
+- And obviously lot of ❤️
+
 
 
-## Project Structure
+<!-- Project Breakdown -->
+## Project Architecture 
+
 - `backend` folder contains the code for the Django based API to upload the files into the projects directory.
 - `code_converter` folder contains the frontend code deployed on DFINITY.
 - `cpp-pseudogen` folder contains the translator code for cpp/c languages.
 - `frontend` folder contains the UI of the entire webapp.
 - `python-scripts` folder contains the translator for python source code.
 
-## How to run the project locally.
+
+
+<!-- GETTING STARTED -->
+
+## Getting Started as Contributor
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+This is an example of how to list things you need to use the software and how to install them.
+
+- npm
+
+```sh
+npm install npm@latest -g
+```
+
+### Installation
+
+1. Clone the repo
+
+```sh
+git clone https://github.com/Open-Sourced-Olaf/Code-Kindle.git
+```
+
+2. Install NPM packages
+
+```sh
+npm install
+```
+3. Run Locally
+
+```sh
+npm run start
+```
+<!-- ROADMAP -->
+
+## Roadmap
+
+- Add support to more languages
+- Convert one language code into other like cpp code to php and so on.
+- Improving the algorithm of the translator to include complex cases like multiple functional calls inside the main.
+- See the [open issues](https://github.com/Open-Sourced-Olaf/Code-Kindle/issues) for a list of proposed features (and known issues).
+
+
+
+<!-- CONTRIBUTING -->
+
+## Local Dev Setup
+
 - To translate the cpp/c codes,move to the cpp-pseudogen folder upload a file named "prog.cpp" and then compile the main.cpp using the command
  `g++ main.cpp -o main` after that execute `./main`, it will display the output in the terminal as well as create a output.text file.
  - To translate the python code, move to the python-scripts folder, upload a file named hello.py, then compile the generate-pseudo.py file using the
```

Comment:
change filename `generate-pseudo.py` to `generatePseudo.py` because in code part ( in the directory ) filename is `generatePseudo.py`  same goes for #L116

Reply:
Done !! :)